fsw/cfe-core/src/inc/cfe_sb.h

```diff
@@ -1358,6 +1367,21 @@ static inline CFE_SB_MsgId_t CFE_SB_ValueToMsgId(CFE_SB_MsgId_Atom_t MsgIdValue)
 {
     return MsgIdValue;
 }
+
+/*****************************************************************************/
+/**
+ * \brief Identifies packet type given message ID
+ *
+ * Provides the packet type associated with the given message ID
+ * as defined by #CFESBPktTypeDefs
+ *
+ * \return Packet type
+ * \retval #CFE_SB_PKTTYPE_CMD     Command packet type
+ * \retval #CFE_SB_PKTTYPE_TLM     Telemetry packet type
+ * \retval #CFE_SB_PKTTYPE_INVALID Invalid/unknown packet type
+ */
+uint8 CFE_SB_GetPktType(CFE_SB_MsgId_t MsgId);
```

Comment:
Note there is no real benefit to using a smaller size than 32 bits in most cases, as we generally don't target CPUs under 32 bits.  

When comparing to a literal value the compiler will implicitly expand this return value to `int` when evaluating the equality test, so using the smaller `uint8` type may generate a few _more_ machine instructions when comparing this value (depends on arch and optimization).